assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -3,6 +3,18 @@
     <span>Current log from {{start}}&nbsp;</span>
     <span ng-if="end">to {{end}}</span>
   </div>
+
+  <div row main-axis="end" ng-if="canViewArchive">
+    <form
+      ng-if="canViewArchive"
+      action="{{kibanaAuthUrl}}"
+      method="POST" target="_blank">
```

Comment:
I tend to think anytime you navigate to a new app (even though we have Kibana somewhat skinned) it should open a new tab... I'd rather drop the back link.